Fix the bipod's scatter and slow values.

Arendian opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
The bipod now doesn't applies multiple stacking slows anymore, only a slow and a  superslow.(which don't stack)

The bipod now also refreshes the burst scatter multiplier when deployed or undeployed, this means that the scatter reduction when using a deployed bipod is now properly applied. Previously it would only apply if the weapon had at least one other attachment that would refresh the burst scatter multiplier, and if it was deployed before wielding.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Parity and bug fix.
Resolves #6761

## Technical details
<!-- Summary of code changes for easier review. -->
Previously a wielded M54CE2 with only a bipod would have a max scatter of 90 degrees in automatic mode with the bipod deployed and no other attachments, this is now 50 degrees because the burst scatter value gets refreshed upon deploying/undeploying the bipod.

If the M54CE2 was used with a bipod AND reflex sight, it would only apply the bipod scatter reduction if it was first deployed and then wielded, this is because the reflex sight would call a refresh for burst scatter multiplier upon wielding the gun. If the gun was first wielded(refresh burst scatter multiplier because of reflex) and then deployed the bipod, the scatter wouldn't change at all because the action of deploying the bipod would not refresh the burst scatter value.

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [X] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl: Dygon
- fix: Improperly folding the bipod attachment now applies a weaker slowdown effect.
- fix: The bipod attachment now correctly applies it's scatter reduction effect when toggled.

Vermidia commented:
I was unware temporary speed mods were a thing. I already added the speed mods on undeploy through the method BreakSlow.

Arendian commented:
> I was unware temporary speed mods were a thing. I already added the speed mods on undeploy through the method BreakSlow (Which are the same values but doubled because remember CM handling status effect results in the numbers x2 90% of the time).

I forgot about the doubling also counting for slows. I removed the method you made since the already existing component allows for more flexible slow durations.
